exonum/src/api/node/public/explorer.rs

```diff
@@ -156,8 +164,45 @@ impl ExplorerApi {
             })
     }
 
+    /// Subscribes to block commits events.
+    pub fn handle_subscribe(
+        name: &'static str,
+        backend: &mut actix::ApiBuilder,
+        shared_api_state: SharedNodeState,
+    ) {
+        let server_address = Arc::new(Mutex::new(None));
+
+        let index = move |req: HttpRequest| -> FutureResponse {
+            let server = server_address.clone();
+            let mut address = server.lock().expect("Expected mutex lock");
+            if address.is_none() {
+                *address = Some(Arbiter::start(|_| WsServer::default()));
+
+                shared_api_state.set_server_address(address.to_owned().unwrap());
+            }
+
+            let _ = ws::start(req.clone(), WsSession::new(address.to_owned().unwrap()));
+
+            Box::new(Ok(HttpResponse::Ok().json(())).into_future())
```

Comment:
Result of this closure is ignored?
Why you translate HttpResponse into json?
Can we just return empty tuple?

Reply:
Closure must have `fn(HttpRequest) -> FutureResponse` signature because we convert it to `Arc<RawHandler>` a bit later